ansible_runner/utils.py

```diff
@@ -188,8 +188,10 @@ def dump_artifacts(kwargs):
         if isinstance(obj, Mapping):
             kwargs['inventory'] = dump_artifact(json.dumps(obj), path, 'hosts.json')
         elif isinstance(obj, string_types):
-            if not os.path.exists(obj):
+            if not os.path.exists(os.path.join(path,obj)):
                 kwargs['inventory'] = dump_artifact(obj, path, 'hosts')
+            else:
+                kwargs['inventory'] = os.path.join(path,obj)
```

Comment:
Since it was previously possible to use this by providing an absolute path, should we maybe use a fallback where we treat it as an absolute path? Maybe not exactly as a fallback, but depending on what `os.path.isabs` gives.